src/karapace/api/oidc/middleware.py

```diff
@@ -120,6 +123,7 @@ def authorize_request(self, payload: dict, request_method: str) -> bool:
                 allowed_roles,
             )
             raise HTTPException(status_code=403, detail="Insufficient roles")
+        log.debug("Authorized")
```

Comment:
this also?

Reply:
Is it ok if we remove these after aiven core integration, in the final steps ? These help in development.

Reply:
Ah yeah, will change these to info for now, as I need them.

Reply:
I think debug is fine as you only need them during development.